### PR TITLE
Fix: doc: Properly substitute PCMK__REMOTE_SCHEMA_DIR into docs.

### PR DESCRIPTION
**Pull request by clumens**

### CODE REVIEW

File: doc/sphinx/conf.py.in [hunk 1]
```diff
@@ -41,6 +41,7 @@ rst_prolog="""
 .. |PCMK_INIT_ENV_FILE| replace:: ``%PACEMAKER_CONFIG_DIR%/pcmk-init.env``
 .. |PCMK_LOG_FILE| replace:: %CRM_LOG_DIR%/pacemaker.log
 .. |PCMK_CONTAINER_LOG_FILE| replace:: ``/var/log/pcmk-init.log``
+.. |PCMK__REMOTE_SCHEMA_DIR| replace:: %PCMK__REMOTE_SCHEMA_DIR%
```
Comment (nrwahl2) [inline doc/sphinx/conf.py.in:44]: I wonder what happened, because I just did this a week ago.

Comment (nrwahl2) [reply doc/sphinx/conf.py.in:44]: Either way thanks for fixing it :)

Comment (nrwahl2) [reply doc/sphinx/conf.py.in:44]: Uhh https://github.com/ClusterLabs/pacemaker/commit/e4e0ed64ae6a2709883097a8059a49ed30869fcd

**Comment by clumens** [comment]
Gotta git pull first, I guess.  Also this will need to go into the appropriate RHEL builds.

**Pull request closed without merging**